### PR TITLE
Fix MBTF issues for 2.2

### PR DESCRIPTION
**Pull request by ShakoHo**
change list--
add camera apps, add sleep after press to fix the tap switch issue
add music app.py to fix the scroll checking
modify test case broswer navigation, camera, music

please help to review, @ypwalter r?

### CODE REVIEW

File: mtbf_driver/mtbf_apps/music/app.py [hunk 1]
```diff
@@ -1,11 +1,26 @@
...
-
+from marionette import expected
+from marionette import Wait
 from marionette.by import By
+
 from gaiatest.apps.music.app import Music
 
+from mtbf_driver.mtbf_apps.music.regions.list_view import Mtbf_ListView
+
 class Mtbf_Music(Music):
```
Comment (ypwalter) [inline mtbf_driver/mtbf_apps/music/app.py]: app.py:10:1: E302 expected 2 blank lines, found 1

File: mtbf_driver/mtbf_apps/music/app.py [hunk 2]
```diff
@@ -1,11 +1,26 @@
...
     def back_from_playback(self):
         self.marionette.find_element(*self._header_locator).tap(25, 25)
         self.wait_for_element_displayed(*self._music_tiles_locator)
+
+    def tap_songs_tab(self):
+        element = Wait(self.marionette).until(
+            expected.element_present(*self._songs_tab_locator))
+        Wait(self.marionette).until(expected.element_displayed(element))
+        element.tap()
+        if self.marionette.find_element(*self._top_bannder_locator).text <> 'Songs':
```
Comment (ypwalter) [inline mtbf_driver/mtbf_apps/music/app.py]: app.py:23:74: W603 '<>' is deprecated, use '!='

File: mtbf_driver/mtbf_apps/music/app.py [hunk 3]
```diff
@@ -1,11 +1,26 @@
...
+            return Mtbf_ListView(self.marionette)
+        else:
+            return Mtbf_ListView(self.marionette,bolScrollingCheck=False)
```
Comment (ypwalter) [inline mtbf_driver/mtbf_apps/music/app.py]: app.py:26:49: E231 missing whitespace after ','

File: mtbf_driver/mtbf_apps/music/regions/list_view.py [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+
+from marionette import Wait
+
+
+from gaiatest.apps.base import Base
+from gaiatest.apps.music.regions.list_view import ListView
+
+class Mtbf_ListView(ListView):
```
Comment (ypwalter) [inline mtbf_driver/mtbf_apps/music/regions/list_view.py]: list_view.py:12:1: E302 expected 2 blank lines, found 1

File: mtbf_driver/mtbf_apps/music/regions/list_view.py [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+
+
+    def __init__(self, marionette, bolScrollingCheck = True):
```
Comment (ypwalter) [inline mtbf_driver/mtbf_apps/music/regions/list_view.py]: list_view.py:15:5: E303 too many blank lines (2)
list_view.py:15:53: E251 no spaces around keyword / parameter equals

File: mtbf_driver/mtbf_apps/music/regions/list_view.py [hunk 6]
```diff
@@ -0,0 +1,22 @@
...
+        Base.__init__(self, marionette)
+        if bolScrollingCheck:
+            Wait(self.marionette).until(
+                lambda m: self.marionette.find_element(*self._view_locator).get_attribute('class') == 'scrolling')
+        Wait(self.marionette).until(
+            lambda m: self.marionette.find_element(*self._view_locator).get_attribute('class') != 'scrolling')
+
```
Comment (ypwalter) [inline mtbf_driver/mtbf_apps/music/regions/list_view.py]: list_view.py:23:1: W391 blank line at end of file

File: mtbf_driver/mtbf_apps/music/app.py (hunk 1 above)
Comment (ShakoHo) [inline mtbf_driver/mtbf_apps/music/app.py]: fixed

File: mtbf_driver/mtbf_apps/music/app.py (hunk 2 above)
Comment (ShakoHo) [inline mtbf_driver/mtbf_apps/music/app.py]: fixed

File: mtbf_driver/mtbf_apps/music/app.py (hunk 3 above)
Comment (ShakoHo) [inline mtbf_driver/mtbf_apps/music/app.py]: fixed

File: mtbf_driver/mtbf_apps/music/regions/list_view.py (hunk 4 above)
Comment (ShakoHo) [inline mtbf_driver/mtbf_apps/music/regions/list_view.py]: fixed

File: mtbf_driver/mtbf_apps/music/regions/list_view.py (hunk 5 above)
Comment (ShakoHo) [inline mtbf_driver/mtbf_apps/music/regions/list_view.py]: fixed

File: mtbf_driver/mtbf_apps/music/regions/list_view.py (hunk 6 above)
Comment (ShakoHo) [inline mtbf_driver/mtbf_apps/music/regions/list_view.py]: fixed

**Comment by ypwalter** [comment]
I have no further concern as for now. thanks! r+

**Comment by zapion** [comment]
For music app, I will try to feedback mtbf_app changes so that we can get rid of this wrapped class

**Pull request merged**